### ISSUE TITLE
Is there documentation on the algorithm?

### ISSUE THREAD

**Issue by Galto2000** [jarikomppa/proctree#4]
Greetings!

Is there by any chance documentation or references available that explains the algorithm being used?

Thanks in advance!

G

**Comment by jarikomppa** [comment]
No, and in addition the author of the original doesn't remember how the
code works either.
